api/queries_pr.go

```diff
@@ -1149,6 +1152,14 @@ func BranchDeleteRemote(client *Client, repo ghrepo.Interface, branch string) er
 	return client.REST(repo.RepoHost(), "DELETE", path, nil, nil)
 }
 
+// sortPRsByPrecedence ensures that OPEN PRs precede non-open states (MERGED, CLOSED)
+func sortPRsByPrecedence(prs []PullRequest) []PullRequest {
```

Comment:
One small nit is that can we name this `sortPullRequestsByState`?

Reply:
of course 😁